packages/next-templates/src/components/TopTask/TopTaskHeader.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2024 Gemeentevoorbeeld.nl
+ * Copyright (c) 2024 Ryan van Valkenhoef
+ */
+
+import clsx from 'clsx';
+import { HTMLAttributes, ForwardedRef, forwardRef, PropsWithChildren } from 'react';
+import './TopTaskHeader.css';
+
+export type DivProps = HTMLAttributes<HTMLDivElement>;
```

Comment:
`DivProps` lijkt me geen handige publieke interface-naam, want die zou niet meer kloppen als je extra properties toevoegt. Dan is het een breaking change om 'm weer aan te passen. Waarom niet gewoon `TopTaskHeaderProps`?